.travis.yml

```diff
@@ -2,27 +2,32 @@ language: php
 
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 git:
-    depth: 1
+  depth: 1
 
 services:
   - mysql
+  
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
 
 matrix:
-    include:
-        - php: hhvm
-        - php: 5.5
-        - php: 5.6
-          env: deps=high
```

Comment:
you didn't use any of these special env variables, so defining a matrix isn't needed (you can just use the `php` section)